modules/living_spaces_default/living_spaces_default.module

```diff
@@ -47,6 +47,22 @@ function living_spaces_default_form_group_form_alter(array &$form, FormStateInte
   $form['is_default']['#access'] = \Drupal::currentUser()->hasPermission('manage living spaces default content');
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function living_spaces_default_form_living_spaces_group_membership_form_alter(array &$form, FormStateInterface $form_state, $form_id) {
+  $info = $form_state->getBuildInfo();
+
+  if (!empty($info['args'][0]) && $info['args'][0] instanceof GroupInterface) {
+    /** @var \Drupal\group\Entity\GroupInterface $group */
+    $group = $info['args'][0];
+
+    if ($group->get('is_default')->getString()) {
+      $form['#access'] = FALSE;
```

Comment:
It looks like this will just show empty page for user without even telling why.
Please implement entity constraint instead which error message that says "Space templates cannot be joined"